ui/common/src/common.ts

```diff
@@ -53,6 +53,47 @@ export const toggle = (initialValue: boolean, effect: (value: boolean) => void =
   return prop;
 };
 
+export interface Selectable {
```

Comment:
looks like (almost) duplication of selector.ts?

Reply:
Ugh, that's a leftover